example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -22,10 +23,17 @@
 #define GEMM_PIPELINE ck_tile::GemmPipelineAgBgCrMem
 #define UNIVERSAL_GEMM_PIPELINE ck_tile::BaseGemmPipelineAgBgCrMem
 #define GEMM_PIPELINE_SCHEDULER ck_tile::GemmPipelineScheduler::Interwave
+constexpr bool isDoubleSmemBuffer = false;
```

Comment:
Would align better with other parameters naming. 
```suggestion
constexpr bool DoubleSmemBuffer = false;
```